specs/shell/list_processes_at_port.yaml

```diff
@@ -0,0 +1,12 @@
+name: List processes at port
+description: List processes at a port to see if there are running processes.
+author: Maurice Gerhardt
+author_url: https://github.com/heymage
+tags: ["macos", "shell"]
+shells:
+  - zsh
+  - bash
```

Comment:
nit: could we omit the `shells` field both here and above? Since `lsof` is not a shell builtin, it should work in all shells (including Fish)

Reply:
Hey @alokedesai, yes of course! I wasn't sure tbh. Will do change it 👍🏻